{{cookiecutter.project_slug}}/k8s/prod/kustomization.yaml

```diff
@@ -3,5 +3,3 @@ kind: Kustomization
 
 bases:
  - ../base
-resources:
-- ./postgres.cnpg.yaml
```

Comment:
Why is the removal of `postgres.cnpg.yaml` included in the PR? I think it belongs in a separate PR.